app/helpers/notification_count.rb

```diff
@@ -2,12 +2,17 @@ module ExercismWeb
   module Helpers
     module NotificationCount
       def notification_count
```

Comment:
I want to rip rubocop out of the project. It's the wrong thing to worry about these things right now.